src/h5web/visualizations/index.test.ts

```diff
@@ -211,13 +211,16 @@ describe('Visualization definitions', () => {
     });
 
     it('should support group with absolute `default` path to supported NXdata group', () => {
-      const group = makeMyNxEntityGroup('foo', 'NXentry', {
-        defaultPath: '/foo/bar',
-        children: [makeMyNxDataGroup('bar', spectrumDatasetInt1D)],
-        parents: [makeMyNxEntityGroup('root', 'NXroot')],
+      const group = makeMyNxEntityGroup('root', 'NXroot', {
+        children: [
+          makeMyNxEntityGroup('foo', 'NXentry', {
+            defaultPath: '/foo/bar',
+            children: [makeMyNxDataGroup('bar', spectrumDatasetInt1D)],
+          }),
+        ],
       });
 
-      expect(supportsEntity(group)).toBe(true);
+      expect(supportsEntity(group.children[0])).toBe(true);
```

Comment:
It is not as clear that the test should check that it is the `foo` group that should be supported.

Perhaps renaming `group` to `rootGroup` would help.

Reply:
I've also added `const [groupWithDefault] = rootGroup.children;`